src/http/client.rs

```diff
@@ -59,7 +59,23 @@ impl StacksClient {
             .await
     }
 
-    pub async fn create_entity(
```

Comment:
These diffs look bigger than they are --- it's just a rename of `create_entity` -> `register` and then moving the function lower in the file to preserve lexicographic order of functions.